plat/common/include/x86/tls.h

```diff
@@ -33,13 +33,8 @@
 #include <x86/cpu.h> /* rdmsrl, wrmsrl */
 #include <uk/arch/lcpu.h>
 
-/* TODO: On latest CPUs a faster instruction can be used:
- * https://www.kernel.org/doc/html/latest/x86/x86_64/fsgs.html
```

Comment:
TODO: Add comment to give a heads up that this will be moved in the context of plat re-arch.